test/description-render-test.js

```diff
@@ -5,6 +5,102 @@ var assert = require('chai').assert;
 var markdownRenderer = require('../lib/index.js');
 
 describe('Markdown', function() {
+  describe('#parse', function() {
```

Comment:
pardon me, but why is this `parse` function tested in `description-render` tests?
Shouldn't it be moved to its own file `parse-test.js` or `low-level-api-test.js`?